common/STcommon.py

```diff
@@ -31,12 +67,30 @@ def parseArgs():
 	'''Parses args using the argparse lib'''
 	parser = argparse.ArgumentParser(description='Location logging server')
 
+	parser.add_argument('-a', '--address', metavar='ADDRESS')
+	parser.add_argument('-c', '--configure', action='store_true')
 	parser.add_argument('-g', '--generate-keys', metavar='PATH', type=str)
+	parser.add_argument('-L', '--logfile', metavar='LOG_FILE')
```

Comment:
Thank makes sense, but I am not a fan of diverging into upper case letters yet. Would it make sense to change it to something like `-o --output` or `-f --file`? I realize those make it a little less readable, but they are just thoughts.